### PR TITLE
[markdown/zh-cn] Update Chinese translation

### PR DESCRIPTION
**Pull request by mengzhongshi**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by divayprakash** [comment]
@mengzhongshi @inkydragon Is this ready to be merged?

File: zh-cn/markdown-cn.html.markdown [hunk 1]
```diff
@@ -102,139 +132,192 @@ __此文本也是__
...
+```md
 [去 music](/music/).
+```
 
-<!-- Markdown同样支持引用样式的链接 -->
+Markdown同样支持引用形式的链接
 
-[点此链接][link1]以获取更多信息！
-[看一看这个链接][foobar] 如果你愿意的话.
+<pre><code class="highlight">&#x5b;<span class="nv">点此链接</span>][<span class="ss">link1</span>] 以获取更多信息！
```
Comment (inkydragon) [inline zh-cn/markdown-cn.html.markdown]: What's this! I think is no need to use HTML tag, ` `` ` is enough.

**Comment by divayprakash** [comment]
@mengzhongshi Any progress on this?

**Comment by divayprakash** [comment]
@inkydragon Fixed!

**Pull request merged**